website/docs/typescript-support.md

```diff
@@ -5,37 +5,22 @@ title: TypeScript Support
 
 ## Setup
 
-Docusaurus supports writing and using TypeScript theme components. To start using TypeScript, add `@docusaurus/module-type-aliases` to your project:
+Docusaurus supports writing and using TypeScript theme components. To start using TypeScript, add `@docusaurus/module-type-aliases` and some `@types` dependencies to your project:
 
 ```bash
-npm install --save-dev typescript @docusaurus/module-type-aliases
+npm install --save-dev typescript @docusaurus/module-type-aliases @types/react @types/react-router-dom @types/react-helmet
 ```
```

Comment:
Which is why this is now longer

Reply:
I think you need to add `@tsconfig/docusaurus` to this list as well.